portal/templates/admin/admin_base.html

```diff
@@ -1,8 +1,8 @@
 {%- extends "layout.html" -%}
 {% block additional_scripts %}
 <script src="https://cdnjs.cloudflare.com/ajax/libs/jasny-bootstrap/3.1.3/js/jasny-bootstrap.min.js"></script>
+<script src="{{ url_for('static', filename='js/dist/bootstrapTableExtensions.bundle.js') }}"></script>
```

Comment:
naive question: are there any potential conflicts between this and the js file in line 8?

Reply:
Ah, never mind - I see that line 8 is for datatables !